Assets/MixedRealityToolkit/Definitions/Devices/MixedRealityControllerMapping.cs

```diff
@@ -147,10 +145,60 @@ internal void SetDefaultInteractionMapping(bool overwrite = false)
             }
         }
 
+        internal bool UpdateInteractionSettingsFromDefault()
```

Comment:
This code is now effectively duplicated here, any way to unify it?

Reply:
Refactored out `GetDefaultInteractionMappings()`